src/frameworks/compose/discover/index.ts

```diff
@@ -1,26 +1,52 @@
-import { AppSpec } from "../interfaces";
+import { Runtime, FileSystem, FrameworkSpec, RuntimeSpec, FrameworkHooks } from "./types";
+import { NodejsRuntime } from "./runtime/node";
+import { FirebaseError } from "../../../error";
+import { AppBundle } from "../interfaces";
 
 /**
  * Discover framework in the given project directory
  */
-export function discover(): AppSpec {
-  return {
-    baseImage: "us-docker.pkg.dev/firestack-build/test/run:latest",
-    environmentVariables: {
-      NODE_ENV: "PRODUCTION",
-    },
-    installCommand: "npm install",
-    buildCommand: "npm run build",
-    startCommand: "npm run start",
+const availableRuntimes: Runtime[] = [new NodejsRuntime()];
+
+/**
+ *
+ */
```

Comment:
nit* Looks like you accidentally moved the docstring for discover to `availableRuntimes`

Reply:
Oh yes, may be due `Auto Fix all` suggestion